migrations/Version20250610130537.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20250610130537 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql(<<<'SQL'
+            ALTER TABLE caf_evt ADD auto_accept TINYINT(1) DEFAULT 0 NOT NULL
```

Comment:
pkoi pas un boolean ?

Reply:
c'est pas changer le comportant, mais si mysql supporte les 2 termes (`MySQL offers BOOLEAN or BOOL as the synonym for TINYINT(1)`), autant utiliser celui qui est le plus parlant non ?
Après, je veux pas en parler des heures, ca me semblait obvious, mais on laisse comme ca si c'est pas le cas.